debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/ThreadsCommand.java

```diff
@@ -305,4 +316,48 @@ private void displayThreads(PrintStream out) throws DDRInteractiveCommandExcepti
 		}
 	}
 
+	private void displayVirtualThreads(PrintStream out) throws DDRInteractiveCommandException {
+		try {
+			J9JavaVMPointer vm = J9RASHelper.getVM(DataType.getJ9RASPointer());
+			int javaVersion = vm.j2seVersion().bitAnd(JavaVersionHelper.J2SE_SERVICE_RELEASE_MASK).intValue() >> JavaVersionHelper.J2SE_JAVA_SPEC_VERSION_SHIFT;
+			if (javaVersion < 19) {
+				throw new UnsupportedOperationException("Virtual thread is not available before Java19.");
+			}
+			try {
+				Class<J9JavaVMPointer> vmClass = J9JavaVMPointer.class;
+				Lookup lookup = MethodHandles.lookup();
+				MethodHandle liveVirtualThreadListMH = lookup.findVirtual(vmClass, "liveVirtualThreadList", MethodType.methodType(PointerPointer.class));
+				PointerPointer mainVirtualThread = (PointerPointer)liveVirtualThreadListMH.invoke(vm);
+
+				MethodHandle virtualThreadLinkNextOffsetMH = lookup.findVirtual(vmClass, "virtualThreadLinkNextOffset", MethodType.methodType(UDATA.class));
+				UDATA linkNextOffset = (UDATA)virtualThreadLinkNextOffsetMH.invoke(vm);
+				J9ObjectPointer root = J9ObjectPointer.cast(mainVirtualThread.at(0));
```

Comment:
root is lazy initialized, so need to check if it is null before converting to `J9ObjectPointer`